app/models/katello/content_view_docker_filter_rule.rb

```diff
@@ -0,0 +1,23 @@
+module Katello
+  class ContentViewDockerFilterRule < Katello::Model
+    self.include_root_in_json = false
+
+    belongs_to :filter,
+               :class_name => "Katello::ContentViewDockerFilter",
+               :inverse_of => :docker_rules,
+               :foreign_key => :content_view_filter_id
+
+    validates_lengths_from_database
```

Comment:
Currently, a user can enter a tag name that does not exist.  Would it be worthwhile to validate that user provides a valid value?